cmd/create/idp/htpasswd.go

```diff
@@ -304,3 +314,19 @@ func parseHtpasswordFile(usersList *map[string]string, filePath string) error {
 
 	return nil
 }
+
+func validateUserPasswd(username, password string) error {
```

Comment:
NP can we rename this to something like
validateHtUsernameAndPassword?

Reply:
changed